### ISSUE TITLE
Verify and publish new REP token contract on Etherscan

### ISSUE THREAD

**Issue by dmdque** [AugurProject/rep-migration#9]
The new REP token contract doesn't have verified code on [Etherscan](https://etherscan.io/address/0x1985365e9f78359a9B6AD760e32412f4a445E862#code).

Doing so will help make development with REP easier.

**Comment by MicahZoltu** [comment]
Unfortunately EtherScan cannot verify contracts instantiated by a contract, so we can't complete this until EtherScan adds support for that.

**Comment by dmdque** [comment]
Ah interesting, can you point me to the source code for that factory contract? It doesn't look like it's the RepToken in this repo.

**Comment by moeadham** [comment]
Please update this here:
https://etherscan.io/verifyContract?a=0x1985365e9f78359a9B6AD760e32412f4a445E862

**Comment by ghost** [comment]
Still not sure if this is possible re: Etherscan. In the meantime, you can manually verify the REP token contract following the info in the link below: 

https://github.com/AugurProject/augur-core#verifying-the-rep-contract-on-the-main-ethereum-network

**Comment by MicahZoltu** [comment]
@moeadham Unfortunately that doesn't work for contracts created by other contracts.  Etherscan needs to make changes on its end (feature request for them) before we can get Verified on Etherscan working.

@dmdque Does the link @tomkysar supplied answer your question?

**Comment by moeadham** [comment]
Interesting @MicahZoltu . Can you send me a link to the ABI, that might help others as well.

**Comment by dmdque** [comment]
@MicahZoltu @tomkysar yes that answers it, thanks.

**Comment by MicahZoltu** [comment]
@moeadham Sorry, I missed your message.  @nuevoalex knows where the ABI lives, I think it sits in an NPM module somewhere.  There has been talk about publishing it separately, but it is a fairly low priority.  I want to say that this repo has an ABI file in it somewhere, you may want to grab the tarball of that and check: https://www.npmjs.com/package/augur-core

**Comment by dmdque** [comment]
I did it.

![image](https://user-images.githubusercontent.com/5205766/49619741-bd297980-f972-11e8-8534-e8a2caacdf22.png)
